src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -2954,8 +2959,18 @@ private void populateCompileArtifactMap( Map<String, Artifact> compileArtifactMa
      */
     private String getBottomText()
     {
-        int currentYear = Calendar.getInstance().get( Calendar.YEAR );
-        String year = String.valueOf( currentYear );
+        final String year;
+        String buildTime = project.getProperties().getProperty( "project.build.outputTimestamp" );
+        if ( buildTime != null )
+        {
+            year = String.valueOf( DateTimeFormatter.ISO_DATE_TIME.parse( buildTime ).get( ChronoField.YEAR ) );
+        }
+        else
+        {
+            getLog().debug( "Missing property project.build.outputTimestamp, using current year instead" );
```

Comment:
"missing" sounds like a warning. Maybe instead use "property xyz is not set, using..."?

Reply:
Since when did this plugin jump on Java 8?

Reply:
@michael-o https://issues.apache.org/jira/browse/MJAVADOC-675
@bmarwell Let me rephrase it by flipping the statements: if available ... otherwise ...